### PR TITLE
merge Develop2 in develop

### PR DESCRIPTION
**Pull request by zapatam671**
Please review to merge develop2 into develop.

Further information can be found in the recent email I sent

### CODE REVIEW

File: src/js/main.js [hunk 1]
```diff
@@ -154,7 +154,7 @@ fetch('./src/data/STREAM_GAGES_USED.json')
   .then(geojson => {
     const getInfo = (feature, layer) => {
         layer.bindTooltip('USGS Stream Gage', {permanent: true, direction: 'bottom', offset: [0,10], className: 'usgs-stream-gages-tooltip'})
-        layer.bindPopup(`<span align="center"><b>${feature.properties.AGENCY}</b> <br>Stream Gage #: ${feature.properties.GAGE_NUMBE} <br><i>${feature.properties.NAME}</i></span>`);
+        layer.bindPopup(`<span align="center"><b>${feature.properties.AGENCY}</b> <br>Stream Gage #: ${feature.properties.GAGE_NUMBE} <br><i><a href=${feature.properties.USGS_link} >${feature.properties.NAME}</i></a></span>`);
```
Comment (dannikate) [inline src/js/main.js]: @zapatam671 Can you make the link open in a new tab? Currently, it opens in the same tab as the map

**Review by dannikate — CHANGES_REQUESTED** [review]
Please see the comment I left on **Line 157**, where you added the USGS links to the stream gage name

File: src/js/main.js (hunk 1 above)
Comment (zapatam671) [reply src/js/main.js]: I have made the changes and pushed a commit into develop2

**Review by dannikate — APPROVED** [review]
Looks good! Please merge these changes to develop and discard this branch @zapatam671 , thanks!

**Pull request merged**